### DISCUSSION TITLE
Set query parameters on response

### DISCUSSION THREAD

**Discussion by Haberkamp** [remix-run/remix#3050] [Q&A]
Is it possible to set query parameters on the response of the LoaderFunction? What I want to to is to set query parameters for pagination (limit & page) to the browsers url when none are already provided.

The users visits: "/products" and gets "/products?page=1&limit=25"

**Comment by kiliman** [comment]
You can redirect from your loader
```ts
export function loader() {
  return redirect('/products?page=1&limit=25')
}
```
